packages/backend/src/core/RoleService.ts

```diff
@@ -115,6 +131,10 @@ export class RoleService implements OnApplicationShutdown {
 		this.redisForSub.on('message', this.onMessage);
 	}
 
+	async onModuleInit() {
+		this.notificationService = this.moduleRef.get(NotificationService.name);
```

Comment:
気の所為だったら申し訳ないのですが、これで治りませんかね？
```suggestion
		this.notificationService = this.moduleRef.get('NotificationService');
```